scripts/extract-and-replace-translations.ts

```diff
@@ -196,7 +206,7 @@ function extractTranslationsFromCodeFiles(codeFiles: string[]) {
 }
 
 async function getOnlineTranslations() {
-	const response = await fetch(`${process.env.PROXY_URL}/translations/nl.json`, {
+	const response = await fetch(`https://avo2-proxy-qas.hetarchief.be/translations/nl.json`, {
```

Comment:
Should this not be stay an env variable?

Reply:
omdat claire alle vertalingen voor alle environments zal beheren op de ibm qas environment, dus die moeten allemaal van daar hun vertalingen halen. Dus onze locale development moet die ook daar gaan halen.